src/docfx/config/Config.cs

```diff
@@ -336,8 +335,6 @@ private static JArray ToGlobConfigs(JObject obj)
 
             foreach (var (key, value) in obj)
             {
-                if (key.Contains("*"))
```

Comment:
Will JsonUtility handle this error case?

Reply:
no, this check is inconsistent (we don't check every place where folders are misused as glob pattern) and incomplete (glob pattern has many other syntax like `?`, `,`, `|` )